app/policies/group_policy.rb

```diff
@@ -17,14 +17,14 @@ def editable_attributes(all)
     return all if is_admin?
 
     all - %i[members_count leaders_count neighbors_count rules_formatted
-             featured name]
+             featured name pinned_post_id]
```

Comment:
This shouldn't be marked uneditable, admins of the group need to be able to modify it. `is_admin?` is site admins